Xamarin.Forms.Platform.Android/Dispatcher.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	internal class Dispatcher : IDispatcher
```

Comment:
for backward compat, couldn't you make it so that when a dispatcher is not registered, it defaults to Device ?

Reply:
Good idea - that also lets us drop all these duplicate implementations of DispatcherProvider and Dispatcher from the platform projects. Done.